packages/catalog-realm/catalog-app/components/filter-section.gts

```diff
@@ -15,12 +60,24 @@ export default class FilterSection extends GlimmerComponent<FilterSectionArgs> {
       ...attributes
     >
       <section class='filter-group'>
-        {{! Todo: Featured, we can choose to use button or others }}
-        <h3 id='featured-heading' class='filter-heading'>Featured</h3>
-      </section>
+        <h2 class='filter-heading'>
+          Categories
+        </h2>
 
-      <section class='filter-group'>
-        {{! Todo: Category Subtree }}
+        <div class='filter-category-list'>
+          {{#each this.mockCategories as |category|}}
+            <button
+              class={{concat
+                'filter-category-button'
+                (if (eq this.activeCategoryFilterId category.id) ' selected')
+              }}
+              {{on 'click' (fn this.updateCategory category)}}
+              data-test-boxel-filter-category-button={{category.id}}
+            >
+              {{category.name}}
+            </button>
+          {{/each}}
+        </div>
```

Comment:
I think this shud be the component we are creating. The mini-component here and it shud be associated with an onclick here

I dont think there is much need for the bigger FilterSection component